src/content/docs/fr/tutorial/0-introduction/1.mdx

```diff
@@ -0,0 +1,59 @@
+---
+type: tutorial
+title: À propos de ce tutoriel
+description: Comment commencer avec le tutoriel "Créez votre premier blog Astro".
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+
+
+## De quoi ai-je besoin pour commencer ?
+
+Si vous avez quelques connaissances de base en **HTML**, **Markdown**, **CSS** et un peu de **JavaScript**, alors vous êtes prêt(e) à vous lancer ! Vous pourrez suivre l'intégralité du tutoriel en suivant simplement les instructions. Astro convient à tout le monde ! 🧑‍🚀 👩‍🚀 👨‍🚀
+
+Vous aurez également besoin d'un compte [GitHub](https://github.com) (ou similaire) pour publier votre projet sur le web.
+
+<details>
+<summary>Comment utiliser les listes de contrôle en bas de chaque page ?</summary>
+
+Vous les cochez !
+
+À la fin de chaque page, vous trouverez une liste de tâches cliquable que vous devriez maintenant être capable d'accomplir. Cochez ces éléments pour suivre votre progression dans le suivi du tutoriel.
+
+L'utilisation du suivi est facultative, mais elle peut vous aider à vous rappeler où vous en êtes si vous terminez le tutoriel en plusieurs visites. Vous pouvez également laisser certaines listes de contrôle vides pour vous rappeler les unités que vous souhaitez revisiter ultérieurement.
+
+(Ces données sont uniquement enregistrées dans le stockage local de votre navigateur et ne sont pas disponibles ailleurs. Aucune donnée n'est envoyée à Astro ni stockée par lui.)
+</details>
+
+<details>
+<summary>L'Unité 1 concerne des choses que je sais déjà faire. Puis-je la sauter ?</summary>
```

Comment:
`Unité` → `Partie` no? We discussed that on the Discord server with @McFlyPartages